src/queues/job-queue.js

```diff
@@ -1,7 +1,6 @@
 const {newsJob} = require('../processes/news-process');
 const {scoringProcessJob} = require('../processes/scoring-process');
 const {scoringDailyProcessJob} = require('../processes/scoring-daily-process');
-const {deleteActivityProcessJob} = require('../processes/delete-activity-process');
 const {unFollowFeedProcessJob} = require('../processes/unfollow-main-feed');
 const {updateMainFeedBroadProcessJob} = require('../processes/update-main-feed-broad');
 const {syncUserFeedProcessJob} = require('../processes/sync-user-feed');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `deleteActivityProcessJob` has been removed and there's no replacement for it. If `removeActivityProcessJob` is intended to replace `deleteActivityProcessJob`, it should be imported here.
```diff
+ const {removeActivityProcessJob} = require('../processes/remove-activity-process');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->